src/installer/pkg/projects/netcoreapp/src/netcoreapp.depproj

```diff
@@ -70,6 +70,42 @@
     </ItemGroup>
   </Target>
 
+  <!-- Platform manifest needs to include all System*Native items from 3.1 release. This target adds them explicitly,
+      to work around the issue introduced with a fix for https://github.com/dotnet/runtime/issues/33118,
+      Android libraries need to conform to a specific naming convention which necessitated renames of these libraries.
+      For more details of the issue and fix see https://github.com/dotnet/runtime/issues/33450 -->
+
```

Comment:
(Nit) Remove empty line to make doc comment adjacent?

My style personally is this, which incorporates a readability gap:

```xml
  <!--
    Platform manifest needs to include all System*Native items from 3.1 release. This target adds
    them explicitly, to work around the issue introduced with a fix for
    https://github.com/dotnet/runtime/issues/33118, Android libraries need to conform to a specific
    naming convention which necessitated renames of these libraries. For more details of the issue
    and fix see https://github.com/dotnet/runtime/issues/33450
  -->
  <ItemGroup>
```